app/src/main/java/chat/rocket/android/main/presentation/MainPresenter.kt

```diff
@@ -34,20 +37,29 @@ import chat.rocket.android.server.infraestructure.ConnectionManagerFactory
 import chat.rocket.android.server.infraestructure.RocketChatClientFactory
 import chat.rocket.android.server.presentation.CheckServerPresenter
 import chat.rocket.android.util.extension.launchUI
+import chat.rocket.android.util.extension.orFalse
 import chat.rocket.android.util.extensions.adminPanelUrl
 import chat.rocket.android.util.extensions.serverLogoUrl
+import chat.rocket.android.util.retryDB
 import chat.rocket.android.util.retryIO
 import chat.rocket.common.RocketChatAuthException
 import chat.rocket.common.RocketChatException
 import chat.rocket.common.model.UserStatus
 import chat.rocket.common.util.ifNull
 import chat.rocket.core.RocketChatClient
+import chat.rocket.core.internal.rest.createDirectMessage
```

Comment:
is your createDirectMessage function an override of this import?  what is this import?

Reply:
no that is the rest API wrapper that talks to the server

Reply:
I don't understand how this import of createDirectMessage relates to your function defined as createDirectMessage(id: String?).  Which one is being used in the code?  Or am I misunderstanding this entirely?